eq-author/src/components/Collapsible/index.js

```diff
@@ -102,16 +147,30 @@ const Collapsible = ({
   };
 
   return (
-    <Wrapper className={className} data-test="collapsible">
-      <Header className="collapsible-header" data-test="collapsible-header">
-        <Title className="collapsible-title" data-test="collapsible-title">
+    <Wrapper className={className} data-test="collapsible" variant={variant}>
+      <Header
+        className="collapsible-header"
+        data-test="collapsible-header"
+        variant={variant}
+        onClick={
+          variant === "content" && (() => setIsOpen((isOpen) => !isOpen))
```

Comment:
```suggestion
          variant === "content" ? (() => setIsOpen((isOpen) => !isOpen)) : undefined
```

This will stop you from getting console errors when the provided fn evaluates to `false`.

What's the idea with the header vs. the `ToggleCollapsibleButton` being clickable (or not) based on the "default" vs "content" variant? Just curious since I'm not at all familiar with the right hand panel redesign stuff :-)

Reply:
When using default collapsible, we only want the collapsible to close/open when the collapsible text (ToggleCollapsibleButton) itself is clicked. The header is not visible, however extends across the page, so clicking that invisible header without the check would cause the collapsible to open. 

When using the content collapsible, we want the collapsible to close/open when either the header or the text (inside the ToggleCollapsibleButton) is clicked, as the content collapsible's header is visible.

Using that check makes sure that the entire header is clickable if it is a content collapsible, including the text itself. It also makes sure that only the text is clickable when using a default collapsible. Without the check either the header is clickable for default collapsible, or the text is not clickable for the content collapsible (only the header would be clickable).